src/main/java/com/yahoo/bullet/rest/query/PubSubReader.java

```diff
@@ -65,10 +71,13 @@ public void run() {
                         continue;
```

Comment:
subscriber.commit(response.getId()) before continuing?

Reply:
Yes. I guess we should. @akshaisarma, could you confirm this?

Reply:
Yes we should.